default/templates/components/stork/stork-search-css.tpl

```diff
@@ -0,0 +1,18 @@
+<link rel="stylesheet" href="https://files.stork-search.net/releases/v1.5.0/flat.css" />
+<!-- Custom Stork-search styling for Emanote -->
+<style>
+	#stork-search-container {
+		z-index: 1000;
+		background-color: rgb(15 23 42/.8);
+	}
+
+	.stork-overflow-hidden-important {
+		overflow: hidden !important;
+	}
+
+	.stork-wrapper-flat .stork-close-button svg {
+		/* small bugfix in Stork's stock styling */
+		margin: auto;
+	}
+</style>
```

Comment:
@jilleJr Why do we need this file? Can't we do the styling inline (in `stork-search.tpl`, etc.) using Tailwind classes?

Reply:
If there is no reason for it having separate, I'd also inline `stork-search-js.tpl` in `stork-search.tpl`.

Reply:
Sure I can inline them, and I'll try move as much as possible into Tailwind classes. I think the z-index was especially troublesome but I'll give it another go.

Reply:
This can be done in separate PR. Let's get this PR merged right away.